common/config/rush/common-versions.json

```diff
@@ -34,8 +34,8 @@
      * For example, allow some projects to use an older TypeScript compiler
      * (in addition to whatever "usual" version is being used by other projects in the repo):
      */
-    // "typescript": [
```

Comment:
Please leave the commented-out typescript section as an example.

Reply:
Got it, updating

However, I'm seeing following error on `rush update`

```
 ERROR  @azure/amqp-common@1.0.0-preview.6 requires a peer of rhea-promise@^0.1.15 but none was installed.
```